Sources/PackageManagerDocs/Documentation.docc/PackageDescription.md

```diff
@@ -0,0 +1,5 @@
+# Package Description
+
+Create reusable code, organize it in a lightweight way, and share it across your projects and with other developers.
+
+For more information about the `PackageDescription` framework, please visit the [official developer documentation](https://developer.apple.com/documentation/packagedescription).
```

Comment:
```suggestion
For more information about the `PackageDescription` framework, please visit the [API documentation for PackageDescription](https://developer.apple.com/documentation/packagedescription).
```

I think we want to drop "official" here - it's Apple's published documentation, but that phrasing implies to me that these docs aren't "official".